### ISSUE TITLE
Create a visual guide to header/footer/homepage customization

### ISSUE THREAD

**Issue by bricas** [IQSS/dataverse#4775]
Although the documentation provides sample templates for customization, it is unclear where the content in these templates will show up and what they might replace from the normal display. A diagram would be very helpful.

**Comment by pdurbin** [comment]
@bricas good idea. The best visual guide I can think of at the moment the poster by @shlake at the Dataverse Community Meeting last week: https://twitter.com/shlakeuva/status/1006292876525998081

![screen shot 2018-06-22 at 2 23 00 pm](https://user-images.githubusercontent.com/21006/41792753-e6059bbe-7627-11e8-8855-307cf3fff3b9.png)

**Comment by shlake** [comment]
I can help with a document of my "poster" which has my github with my customization files:
[UVaHomePage.pdf](https://github.com/IQSS/dataverse/files/2128735/UVaHomePage.pdf)

**Comment by bricas** [comment]
@shlake This is very helpful, thanks! :+1:

**Comment by pdurbin** [comment]
@shlake yes! Very helpful! I'm just going to link to https://github.com/shlake/LibraDataHomepage from your poster and paste PNG images of all three pages below:

![uvahomepage1](https://user-images.githubusercontent.com/21006/41793814-410b83f4-762b-11e8-9957-2b4ef586de23.png)

![uvahomepage2](https://user-images.githubusercontent.com/21006/41793813-40ef1afc-762b-11e8-8f8b-8b3a96de0df8.png)

![uvahomepage3](https://user-images.githubusercontent.com/21006/41793812-40c4cf04-762b-11e8-81d0-88746b1baeb9.png)

**Comment by pdurbin** [comment]
#4467 is related.

**Comment by pdurbin** [comment]
Yesterday at backlog grooming we decided we'd simply link to the great work by @shlake , her git repo and poster, so I just created pull request #4794 for this. Moving to code review at https://waffle.io/IQSS/dataverse
